lib/dict.ts

```diff
@@ -2,10 +2,12 @@
 export default function () {
     var dict = {};
     var args = arguments
-
+    if(args.length % 2 == 1){
+        throw new Error(`odd dict error`);
```

Comment:
Use this function to throw error: https://github.com/ysmood/nisp/blob/master/core/index.ts#L76